pkg/tfvars/powervs/powervs.go

```diff
@@ -47,20 +36,34 @@ type TFVarsSources struct {
 	APIKey         string
 	SSHKey         string
 	PowerVSRegion  string
+	PowerVSZone    string
+	VPCZone        string
 	CISInstanceCRN string
 }
 
 // TFVars generates Power VS-specific Terraform variables launching the cluster.
 func TFVars(sources TFVarsSources) ([]byte, error) {
 	masterConfig := sources.MasterConfigs[0]
 
+	// TODO(mjturek): Allow user to specify vpcRegion in install config like we're doing for vpcZone
+	vpcRegion := rhcos.PowerVSRegions[sources.PowerVSRegion].VPCRegion
+
+	vpcZone := sources.VPCZone
+	if vpcZone == "" {
+		// Randomly select a zone in the VPC region.
+		rand.Seed(time.Now().UnixNano())
```

Comment:
i like this idea for the sake of not overloading one dc, but i think we should do some latency tests to see which vpc zone is closest to each powervs zone, and create some kind of map for default zone. maybe in a followup pr. thoughts?

Reply:
I like this idea! It will take some research and playing around but worthwhile. I'd prefer to do it as a follow up but up to you